frontend/templates/iframe.ejs

```diff
@@ -130,15 +130,17 @@
       setInterval(function () {
         if (document.body.offsetHeight !== lastHeight && document.body.offsetHeight > 10) {
           lastHeight = document.body.offsetHeight;
-          window.parent.postMessage(JSON.stringify({ remarkIframeHeight: lastHeight }), '*');
+          window.parent.postMessage({ remarkIframeHeight: lastHeight }, '*');
         }
       }, 200);
 
       window.addEventListener('message', receiveMessages);
 
       function receiveMessages(event) {
         try {
-          const data = typeof event.data === 'string' ? JSON.parse(event.data) : event.data;
+          const data = event.data;
+          const isObj = typeof data === 'object' && data !== null && !Array.isArray(data);
```

Comment:
sorry to interfere, but this code is repeated, I think it should be put in a separate function in utils

Reply:
fixed ;)